public/locales/nl/challenge.json

```diff
@@ -0,0 +1,129 @@
+{
+  "INTRODUCTION_TITLE": "Introduction",
```

Comment:
I think NL is the default, so if you only fill it in there, it should work for both.

Reply:
Ah, good to know! Will change this tomorrow!

Reply:
@mdvanes It turned out that there is no fallback. So I'll keep this translation file for now!